distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering.enso

```diff
@@ -128,9 +128,9 @@ type Comparable
        A custom comparator is any comparator that is different than the
        default ones.
     has_custom_comparator : Atom -> Boolean
-    has_custom_comparator atom =
-        comp = Comparable.from atom
-        (comp.is_a Default_Comparator).not
+    has_custom_comparator atom = case Comparable.from atom of
```

Comment:
I'll try to get rid of this function by writing it in Java, but otherwise I think this is ready for your review, gentlemen!

Reply:
Done in [904055d](https://github.com/enso-org/enso/pull/6280/commits/904055d38b2bf48fb3732c07125c6112c0b9ea6e)